app/models/location.rb

```diff
@@ -374,6 +374,7 @@ def available_from
   end
 
   def status
+    return :incomplete if [postcode, units, type_of_unit, mobility_type].any?(&:blank?)
     return :active if deactivation_date.blank?
```

Comment:
@natdeanlewissoftwire I can't find your comment now ("do we need returns here for the other statuses?") - not sure if you deleted it or if it was because I force pushed? Although I'd be surprised if force pushing caused the comment to be deleted.

Anyways, I was trying to keep the scope of this ticket to be just the display of the statuses. Under that logic I arguably didn't need to add this line returning the incomplete status... I think it was because I knew the de/reactivating logic was being done in other tickets but I didn't think the incomplete status was covered elsewhere currently.

Happy to remove. I've added a comment on https://digital.dclg.gov.uk/jira/browse/CLDC-1669 now anyway indicating that the incomplete status logic will need to be updated as part of that ticket.

Reply:
Yep - I deleted as I re-read your description of the PR and saw that you left it out intentionally, I agree that's the right thing to do :D